hcsr04/hcsr04.go

```diff
@@ -0,0 +1,81 @@
+// Package hcsr04 provides a driver for the HC-SR04 ultrasonic distance sensor
+//
+// Datasheet:
+// https://cdn.sparkfun.com/datasheets/Sensors/Proximity/HCSR04.pdf
+package hcsr04
+
+import (
+	"machine"
+	"time"
+)
+
+const TIMEOUT = 23324 // max sensing distance (4m)
+
+// Device holds the pins
+type Device struct {
+	trigger machine.Pin
+	echo    machine.Pin
+}
+
+// New returns a new ultrasonic driver given 2 pins numbers (not pin object)
```

Comment:
> given 2 pins numbers (not pin object)

I'm not sure what to make of that. Is this maybe a comment from before the `machine.GPIO` -> `machine.Pin` transition?

Reply:
Yes, fixed. I fixed too the driver were I copied that from